src/plugins/autonomy/ExternalControl/ExternalControl.cpp

```diff
@@ -46,6 +46,10 @@
 #include <limits>
 
 #include <boost/optional.hpp>
+#include <boost/algorithm/string.hpp>
+
+#include <chrono>
```

Comment:
These are the lines that failed in the cpplint check. You can add ``// NOLINT`` for these since we don't mind using c++11 headers.